parl/utils/utils.py

```diff
@@ -134,10 +135,22 @@ def isnotebook():
 def check_version_for_xpu():
     """check paddle version if the code want to use xpu
     """
-    err = "To use xpu, PARL requires paddle version >= 2.0.0." \
+    err = "To use xpu, PARL requires paddle version >= 2.0.0. " \
           "Please make sure the version is good with your code."
     import paddle
     from paddle import fluid
 
     paddle_version = get_fluid_version()
     assert paddle_version >= 200 or paddle_version == 0, err
+
+
+def check_version_for_fluid():
+    """check paddle version if the code want to use fluid parl
```

Comment:
check the paddle version if the code requires fluid-based implementation.

Reply:
done